### PR TITLE
Responsive settings

### LINKED ISSUE #189: [Feature] responsive options page
**Is your feature request related to a problem? Please describe.**

when i use the options page in a tall window it doesn't resize well (adds scrollbars and is generally funny looking)

**Describe the solution you'd like**

properly make the sidebar smaller based on screensize

**Describe alternatives you've considered**

no

**Additional context**

i know this isn't mobile supported but its more professional if it does this

### PR DESCRIPTION
**Pull request by GrahamSH-LLK**
Adds responsive settings page.
Settings - Scratch Addons
Resolves #189

### CODE REVIEW

File: images/icons/close.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="24" height="24" preserveAspectRatio="xMidYMid meet" viewBox="0 0 24 24" style="-ms-transform: rotate(360deg); -webkit-transform: rotate(360deg); transform: rotate(360deg);"><path d="M8 19a3 3 0 0 1-3-3V8a3 3 0 0 1 3-3a1 1 0 0 0 0-2a5 5 0 0 0-5 5v8a5 5 0 0 0 5 5a1 1 0 0 0 0-2zm7.71-3.29a1 1 0 0 0 0-1.42L13.41 12l2.3-2.29a1 1 0 0 0-1.42-1.42L12 10.59l-2.29-2.3a1 1 0 0 0-1.42 1.42l2.3 2.29l-2.3 2.29a1 1 0 0 0 0 1.42a1 1 0 0 0 1.42 0l2.29-2.3l2.29 2.3a1 1 0 0 0 1.42 0zM16 3a1 1 0 0 0 0 2a3 3 0 0 1 3 3v8a3 3 0 0 1-3 3a1 1 0 0 0 0 2a5 5 0 0 0 5-5V8a5 5 0 0 0-5-5z" fill="#fff"/><rect x="0" y="0" width="24" height="24" fill="rgba(0, 0, 0, 0)" /></svg>
```
Comment (wgyt735yt) [inline images/icons/close.svg:1]: Everything seems great, though I'm not quite sure about the close icon 😐

**Review by wgyt735yt — CHANGES_REQUESTED** [review]
Everything seems great, though I'm not quite sure about the close icon 😐

File: images/icons/close.svg (hunk 1 above)
Comment (GrahamSH-LLK) [reply images/icons/close.svg:1]: I couldn't find a better icon in the iconset.

Comment (wgyt735yt) [reply images/icons/close.svg:1]: @GrahamSH-LLK I understand that, but could I edit the icon?

Comment (GrahamSH-LLK) [reply images/icons/close.svg:1]: Not easily...

Comment (wgyt735yt) [reply images/icons/close.svg:1]: @GrahamSH-LLK I edited it, and here is the code:
`<svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="24" height="24" viewBox="0,0,24,24"><g transform="translate(-228,-168)"><g data-paper-data="{&quot;isPaintingLayer&quot;:true}" fill-rule="nonzero" stroke="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke-miterlimit="10" stroke-dasharray="" stroke-dashoffset="0" style="mix-blend-mode: normal"><path d="M243,184.0058c-0.26664,0 -0.52223,-0.10649 -0.71,-0.2958l-2.29,-2.3l-2.29,2.3c-0.18777,0.18931 -0.44336,0.2958 -0.71,0.2958c-0.26664,0 -0.52223,-0.10649 -0.71,-0.2958c-0.18931,-0.18777 -0.2958,-0.44336 -0.2958,-0.71c0,-0.26664 0.10649,-0.52223 0.2958,-0.71l2.3,-2.29l-2.3,-2.29c-0.39212,-0.39212 -0.39212,-1.02788 0,-1.42c0.39212,-0.39212 1.02788,-0.39212 1.42,0l2.29,2.3l2.29,-2.3c0.39212,-0.39212 1.02788,-0.39212 1.42,0c0.39212,0.39212 0.39212,1.02788 0,1.42l-2.3,2.29l2.3,2.29c0.18931,0.18777 0.2958,0.44336 0.2958,0.71c0,0.26664 -0.10649,0.52223 -0.2958,0.71c-0.18777,0.18931 -0.44336,0.2958 -0.71,0.2958z" fill="#ffffff"/><path d="M228,192v-24h24v24z" fill-opacity="0" fill="#000000"/></g></g></svg>`

**Review by wgyt735yt — APPROVED** [review]
I'm glad the icon was changed, I like how this looks and how it works. I approve.

**Comment by GrahamSH-LLK** [comment]
@WorldLanguages This should be ready to merge!

**Review by WorldLanguages — APPROVED** [review]

**Pull request merged**